nullaway/src/main/java/com/uber/nullaway/NullAway.java

```diff
@@ -974,7 +959,9 @@ private boolean symbolHasSuppressInitalizationWarningsAnnotation(Symbol symbol)
     SuppressWarnings annotation = symbol.getAnnotation(SuppressWarnings.class);
     if (annotation != null) {
       for (String s : annotation.value()) {
-        if (s.equals(INITIALIZATION_CHECK_NAME)) {
+        // we need to check for standard suppressions here also since we may report initialization
+        // errors outside the normal ErrorProne match* methods
+        if (s.equals(INITIALIZATION_CHECK_NAME) || allNames().stream().anyMatch(s::equals)) {
```

Comment:
LGTM. Just, out curiosity, what is `allNames`? I am not sure where that one comes from.

Reply:
Ah, never mind, is all names for this checker based on Error Prone's BugCheckerInfo. All good.

Reply:
`allNames` includes all checker names, which for us is `NullAway` and (legacy) `CheckNullabilityTypes`